helpers/typescript.ts

```diff
@@ -175,7 +175,7 @@ export const installTSTemplate = async ({
 
     // copy loader component
     const dataSourceType = dataSources[0]?.type;
```

Comment:
this doesn't work on an empty array

Reply:
already have a question mark there so it should be ok

Reply:
you're right - wrote to much java code in my life where it would throw an exception :D